src/components/facet/_facet_button.scss

```
@@ -25,7 +25,9 @@
   }
 
   &:focus {
-    background-color: $euiColorLightestShade;
+    background-color: $euiFocusBackgroundColor;
+    // use box-shadow as a "faux outline" to apply left/right padding only
+    box-shadow: -4px 0 $euiFocusBackgroundColor, 4px 0 $euiFocusBackgroundColor;
```

We also have variables for pixel values that you'll want to use
```suggestion
    box-shadow: (-$euiSizeXS) 0 $euiFocusBackgroundColor, $euiSizeXS 0 $euiFocusBackgroundColor;
```